drivers/hmis/app/graphql/types/application/user.rb

```diff
@@ -30,6 +39,26 @@ class Application::User < Types::BaseObject
     field :date_created, GraphQL::Types::ISO8601DateTime, null: false
     field :date_deleted, GraphQL::Types::ISO8601DateTime, null: true
 
+    # audit_history returns the changes this user has made (as opposed to activity_logs which is just views, not edits).
+    # We use the generic term 'audit' to encompass both types of history (view and edit), but many places in the code,
+    # 'audit' just refers to edit history.
+    audit_history_field(
+      excluded_keys: EXCLUDED_KEYS,
+      # filter_args: { type_name: 'UserAuditEvent' },
+      filter_args: { type_name: 'UserAuditEvent', omit: [:user] },
+    )
+
+    def audit_history(filters: nil)
+      v_t = GrdaWarehouse.paper_trail_versions.arel_table
+      scope = GrdaWarehouse.paper_trail_versions.
+        where(user_id: object.id).
```

Comment:
Whoops I think we actually want this to check `true_user_id`.
Actions that a user takes while "impersonating" a different user should show up in the REAL user's user audit history. For example if I impersonate you and create a client, the client creation should show up on my audit history (And not yours).

Does that make sense? To test it out, Admin>Users>Impersonate User. If its not there you need to enable the permission. Thanks!

Reply:
Yes it makes sense, I made the change you suggested and tested it out locally, and it seems to be working correctly.

A new question: While I was testing in local dev, I noticed that the user audit screen isn't recording when someone creates a client alert. Is that expected? I can't see it in the `activity_logs` table either, but I'm not sure if that is the right table to check?

Reply:
Paper Trail is not enabled on the `Hmis::ClientAlert` model, but it would probably be a good idea to add that. I created a ticket https://www.pivotaltracker.com/story/show/187012163 feel free to pick it up if you are looking for work 🙂